### ISSUE TITLE
Cannot specify Azure Environment

### ISSUE THREAD

**Issue by mattfirtion** [pendrica/chef-provisioning-azurerm#25]
When provisioning resources using ARM, the gem does not allow changing the management API URL to use other environment, i.e. Azure China, Azure US Gov, etc.

**Comment by shabiel** [comment]
I hardcoded it this way:

chef-provisioning-azurerm-0.4.0/lib/chef/provisioning/azurerm/azure_provider.rb
```
         def resource_management_client
           credentials = Credentials.new.azure_credentials_for_subscription(new_resource.subscription_id)
           client = Azure::ARM::Resources::ResourceManagementClient.new(credentials, "https://management.usgovcloudapi.net/")
           client.subscription_id = new_resource.subscription_id
           client
         end
```

**Comment by stuartpreston** [comment]
Hi this is fixed in the latest release v0.5.0

You can now provide the driver string as follows:


```
with_driver 'AzureUSGovernment:f3b1e6e5-YOUR-GUID-HERE-abcdabcdabcd'
```

https://github.com/pendrica/chef-provisioning-azurerm#support-for-azureusgovernment-azurechina-azuregermancloud-environments
